plugins/spammer/cpu_usage.go

```diff
@@ -0,0 +1,107 @@
+package spammer
+
+import (
+	"errors"
+	"io/ioutil"
+	"math/rand"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+var (
+	CPUUsageTimePerSample = time.Second / 4
+
+	activeWorkerCount = 0
+
+	once sync.Once
+	mu   sync.Mutex
```

Comment:
please use `syncutils.Mutex`  and give the mutex a name like `cpuUsageErrorLock`

Reply:
Alright. Yes I need to get to know more what is in hive.go . Same goes for sync.Once -> syncutils.Once I assume.

P.S. Why is there a syncutils when in this case it just says?
> type Mutex = sync.Mutex

Reply:
We have that as a placeholder, if we want to enable the "deadlock" detector, we just replace that line with deadlock.Mutex. This way we don't need to change all mutexes in the whole codebase.

Reply:
That is very cool. I'm reading the deadlock package docs...